lib/manager/conan/extract.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { Fixtures } from '../../../test/fixtures';
+import { extractPackageFile } from '.';
+
+const conanfile1 = Fixtures.get('conanfile.txt');
+const conanfile2 = Fixtures.get('conanfile2.txt');
+const conanfile3 = Fixtures.get('conanfile.py');
+
+describe('manager/conan/extract', () => {
+  describe('extractPackageFile', () => {
+    it('returns null for empty', () => {
+      expect(extractPackageFile('nothing here')).toBeNull();
+    });
+    it('extracts multiple image lines from conanfile.txt', () => {
+      const res = extractPackageFile(conanfile1);
+      expect(res?.deps).toMatchSnapshot();
+      expect(res?.deps).toContainValues([
```

Comment:
```suggestion
      expect(res?.deps).toEqual([
```

Shouldn't we use `toEqual` here instead?

Reply:
I think toEqual would expect this to be the full output.  I can put the full output here if that's better.

Reply:
I'm not sure I want to remove any of the deps.  They're all differently formatted

Reply:
That I'm comfortable with.